Task8/README.md

```diff
@@ -0,0 +1,158 @@
+# ZenML pipeline downloading data from BQ in one step and running a zenML pipeline in Vertex AI pipelines
+
+## Notes
+In the folder you will find two python scripts: `pipeline.py` and `run.py`, `pipeline.py` showcases how to download data from BQ in zenML pipeline and 
+`run.py` is used to run another zenML pipeline (mnist) remotely in `Vertex AI` pipelines. The reasons for that is, that so far you can run only pipelines
+that contain python modules installable with `zenml integration install <package>` (list of packages is [here](https://docs.zenml.io/features/integrations)).
+And since to download data from `BigQuery` I used the recommended `pandas_gbq` (not included in zenML integrations) it fails when submitting the job to 
+`Vertex AI` (because zenML defined an image where they install only their integrated packages).
+
+Correction:
```

Comment:
Was this problem why it is was difficult to make it work? Was it easy to follow documentation or what was challenging getting this pipeline working?

Reply:
Yes I would say the main problems were:
- They have some small bugs in their `examples`, and I couldn't figure out where one error was coming from
- In general I found it hard to debug because sometimes the errors are very long and not that informative
- Their documentation requires some background knowledge, which I don't really have, so I needed to figure out some things with trial and error (but revisiting the documentation for the second time I didn't find it that hard to follow it)

Reply:
Can you put this in readme subfolder?

Reply:
I moved the readme to the subfolder